tests/Integration/Resources/ProxyProvider/Test_DIRACCAProxyProvider.py

```diff
@@ -0,0 +1,193 @@
+""" Tests for ProxyProvider modules module
+"""
```

Comment:
Please add this to https://github.com/DIRACGrid/DIRAC/blob/integration/tests/Integration/all_integration_server_tests.sh